index.html

```diff
@@ -4939,18 +4939,57 @@ <h3>Unsigned Claims</h3>
       </section>
 
       <section class="informative">
-        <h3>Token Binding</h3>
+        <h3>Man-in-the-Middle (MITM), Replay, and Cloning Attacks</h3>
+
+        <p>
+The data model does not inherently prevent
+<a href="https://en.wikipedia.org/wiki/Man-in-the-middle_attack">Man-in-the-Middle (MITM)</a>,
+<a href="https://en.wikipedia.org/wiki/Replay_attack">replay</a>, and 
+<a href="https://en.wikipedia.org/wiki/Spoofing_attack">spoofing</a> attacks.
+Both online and offline use cases might be susceptible to these types of attacks,
+where an adversary intercepts, modifies, re-uses, and/or replicates the <a>verifiable
+credential</a> data during transmission or storage. 
+        </p>
+        <h4>Man-in-the-Middle (MITM) Attack</h4>
 
         <p>
 A <a>verifier</a> might need to ensure it is the intended recipient of a
 <a>verifiable presentation</a> and not the target of a
 <a href="https://en.wikipedia.org/wiki/Man-in-the-middle_attack">man-in-the-middle attack</a>.
-Approaches such as token binding [[RFC8471]], which ties the request for a
-<a>verifiable presentation</a> to the response, can secure the protocol.
+Some <a href=#securing-verifiable-credentials>securing mechanisms</a>, 
+like [[VC-JOSE-COSE]] or [[VC-DATA-INTEGRITY]], provide an option to specify 
+the intended audience or domain of a <a>presentation</a>, which can help reduce 
```

Comment:
are there any normative requirements regarding `domain` / `aud`... I am not super comfortable with this sentence.